packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -197,14 +199,16 @@ $Calendar-dayPicker-margin: 10px;
 .dayIsFocused:hover,
 .dayIsUnfocused:hover {
   cursor: pointer;
-  background: $ms-color-neutralLight;
+  background: $ms-color-neutralLighter;
```

Comment:
I believe you also need to make this change in .dayHover class as well

Reply:
Done